webpack-demo/src/index.js

```diff
@@ -51,10 +59,10 @@ const renderTasks = () => {
       trash.style.display = 'block';
     });
 
-    trash.addEventListener('click', () => {
-      deleteTask(index, tasks);
-      renderTasks();
-    });
+    // trash.addEventListener('click', () => {
+    //   deleteTask(index, tasks);
+    //   renderTasks();
+    // });
```

Comment:
- [ ] The delete task method is not working in your project because you commented out this code. kindly consider implementing the delete task method because it was a requirement of 2nd milestone of this project.
another thing to keep in mind is that when I applied these lines of code and remove some tasks, then the remaining tasks' indexes lose their order in the local storage. Kindly consider updating the order on every removal of the task.

https://user-images.githubusercontent.com/96683843/230663798-0d175ba9-ec76-4c72-b61c-770fa36ad3ff.mov